### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by Nathan-MV** [mkxp-z/mkxp-z#80]
It's mainly about what's happening with Struma, and Struma no longer exists

**Comment by JaidenAlemni** [comment]
Not sure if this is a major priority... but since Struma created mkxp-z and contributed to it for years, I will throw my 2 cents in and say that I think it's very important they have some presence in the readme. Be it a formal credit line or whatever. Detailing that they left or why they left is unimportant imo.

As a reminder since it's somewhat related, they also had the wish to keep the icon, as it's the only thing representative of them (and the -z). I think that's a pretty easy thing to fulfill, if everything else is changed.

**Comment by Nathan-MV** [comment]
> Not sure if this is a major priority... but since Struma created mkxp-z and contributed to it for years, I will throw my 2 cents in and say that I think it's very important they have some presence in the readme. Be it a formal credit line or whatever. Detailing that they left or why they left is unimportant imo.
> 
> As a reminder since it's somewhat related, they also had the wish to keep the icon, as it's the only thing representative of them (and the -z). I think that's a pretty easy thing to fulfill, if everything else is changed.

I'm talking about it
![image](https://github.com/mkxp-z/mkxp-z/assets/67521199/def973e0-a7e5-4adc-9534-3e34f720e19f)

**Comment by JaidenAlemni** [comment]
Ah, I see. Couldn't garner that from your initial post. In this case, it makes sense to update it to state that it is being actively managed again.

**Comment by Nathan-MV** [comment]
It's my fault for not providing the context

**Comment by Splendide-Imaginarius** [comment]
Yeah I need to update the README to clarify the current status of the project, just haven't gotten around to it yet.
